### PR TITLE
Audit more models

### PR DESCRIPTION
**Pull request by freddyheppell**
## Overview of Changes

Audit improvements:
- [x] Improvements to boolean display
- [x] Option for fancy and simple old/new diffs
- [x] Relationship diffs

Add audit to models:
- [x] Minecraft players
  - [x] Creation and edits
  - [x] Linking to account
- [x] Servers
  - [x] Creation and edits
- [x] Server ~~keys~~ tokens
  - [x] Creation and edits
- [x] Donations
  - [x] Creation and edits
    * Creation by stripe webhook is labelled
  - [ ] Possibly other events? (@andyksaw )
- [x] Donation perks
- [x] Donation tiers
- [x] Builder rank apps
  - [x] Decisions
- [x] Ban appeals
  - [x] Decisions
- [x] Badges
  - [x] Creation and edits
  - [ ] awarding?
- [x] Groups
  - set up for linking but not create/edit as they're not editable through the panel right now
- [x] Balance transactions

## Deployment Requirements
- [ ] Composer update
- [ ] NPM update
- [ ] Database migration
- [ ] .env update
- [ ] Environment upgrade (eg. PHP version)
- [ ] Other (please specify)

### CODE REVIEW

**Comment by andyksaw** [comment]
@freddyheppell Sorry, can you please merge in the latest `master` branch? Just to be sure it works with the new Vite changes

File: .env.example [hunk 1]
```diff
@@ -4,6 +4,8 @@ APP_KEY=
 APP_DEBUG=true
 APP_URL=http://localhost
 
+MIX_APP_URL="${APP_URL}"
```
[the remark this answers is not in the document]
Comment (freddyheppell) [reply .env.example:7]: Ah yeah, I put that in before the Vite PR actually, just because it was unset on my dev env as it wasn't in the .env.example

**Comment by freddyheppell** [comment]
Merged!

Just a couple of things to bear in mind for future development to make auditing easier:

* if we have to remove an audited property, give me a heads up and I'll add that to the way it works (future me: maybe make a wrapper class which doesn't get added to the list but still provides data if viewing an audit using it, or write a command to purge it from existing audit data
* for actions we want to track but aren't performed by a logged-in user, add it to the `SystemCauser` enum and then set in the command/controller/whatever using `SystemCauseResolver`

**Review by andyksaw — APPROVED** [review]
🚀

**Pull request merged**